glean-core/src/lib.rs

```diff
@@ -306,6 +309,9 @@ fn initialize_inner(
             let upload_enabled = cfg.upload_enabled;
             let trim_data_to_registered_pings = cfg.trim_data_to_registered_pings;
 
+            // Set the logging level for the `log` crate.
+            set_max_level(cfg.log_level.unwrap_or(LevelFilter::Trace));
```

Comment:
We configure different levels for different platforms here: https://github.com/mozilla/glean/blob/306fb0b3d4783360768aaac852ba734945a57cd1/glean-core/src/lib.rs#L630
Default is `Debug`. On iOS release builds we actually limit to `Info`.
Maybe we should only re-set the max level if actually set
```suggestion
            cfg.log_level.map(|level| log::set_max_level(level));
```

(I think not importing the function name directly makes this more readable)